src/core/data/transform.h

```diff
@@ -51,6 +52,16 @@ struct TransformStack : public Transform {
  public:
   void dump() const;
 
+ public:
+  std::vector<int32_t> find_imaginary_dims() const
```

Comment:
missed `override` keyword

Reply:
I believe there should be no override since this method is unique to TrasformStack (has different arguments)